README.md

```diff
@@ -40,6 +18,20 @@ You can use the [GOV.UK Docker environment](https://github.com/alphagov/govuk-do
 
 **Use GOV.UK Docker to run any commands that follow.**
 
+### Building the schemas
+
+```
+bundle exec rake build
+```
+
+### Running the tests
+
+```
+bundle exec rake spec
```

Comment:
```suggestion
bundle exec rake
```

We should encourage people to run the default, as this includes linting (and mirrors what CI will run).

Reply:
I did write that initially, but then adding the "build schemas" section felt odd, as I was specifying commands that are already run as part of the default task. I've tweaked the commit a bit to add this back in, as well as explain how to run the tests in isolation.